### PR TITLE
Daily PR April 25

### PR DESCRIPTION
**Pull request by pe7r**
Working with styles of ProductCard for SliderPopular and slider itself.

### CODE REVIEW

File: src/Components/Sliders/Popular/SliderPopular.js [hunk 1]
```diff
@@ -3,48 +3,71 @@ import {NavLink} from 'react-router-dom'
...
+    next = () => {
+        this.slider.slickNext();
+    }
+
+    previous = () => {
+        this.slider.slickPrev();
+    }
+
+    render() {
+        const {
```
Comment (ruslanpalagin) [inline src/Components/Sliders/Popular/SliderPopular.js]: Please try to follow destructing of props:
const { productsList } = this.props;

File: src/Components/ProductCard/ProductCard.scss [hunk 2]
```diff
@@ -159,6 +159,50 @@
...
+        align-items: center;
+        margin-right: 2px;
+        visibility: hidden;
+    }
+
+    .colors__border--active {
+        display: flex;
+        height: 15px;
+        width: 15px;
+        border-radius: 50%;
```
Comment (ruslanpalagin) [inline src/Components/ProductCard/ProductCard.scss]: please avoid of re-declaring css rules. 
border-radius: 50%; is specified in parent scope (line 80 in this file) as well as many other properties

File: src/Components/Sliders/Popular/SliderPopular.js (hunk 1 above)
Comment (pe7r) [reply src/Components/Sliders/Popular/SliderPopular.js]: Done.

File: src/Components/ProductCard/ProductCard.scss (hunk 2 above)
Comment (pe7r) [reply src/Components/ProductCard/ProductCard.scss]: Done.

**Pull request merged**